api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -129,9 +129,12 @@ message Bootstrap {
   // Optional watchdog configuration.
   Watchdog watchdog = 8;
 
-  // Configuration for an external tracing provider. If not specified, no
-  // tracing will be performed.
-  trace.v3.Tracing tracing = 9;
+  // Configuration for an external tracing provider.
+  //
+  // .. attention::
+  //  This field has been deprecated in favor of :ref:`HttpConnectionManager.Tracing.provider
+  //  <envoy_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.Tracing.provider>`.
+  trace.v3.Tracing tracing = 9 [deprecated = true];
```

Comment:
One question before deprecating, do you have thoughts on https://github.com/envoyproxy/envoy/issues/10576?

Reply:
@htuch Sorry, didn't notice in time.

I've replied [here](https://github.com/envoyproxy/envoy/issues/10576#issuecomment-612126784).